templates/email/order-pickup-ready.hypr

```diff
@@ -224,18 +232,68 @@
             </tr>
         </thead>
         
-    {% for item in model.items %}
-			<tr class="mz-ordersummary-line mz-ordersummary-line-item">
-				<td>
-                {{ item.name }}
-                <dl>
-                    <dd>{{ item.productCode }}</dd>&nbsp;
-                </dl>
-                </td>
-                <td align="right"><span class="mz-item-price">{% filter currency %} {{ item.actualPrice|multiply(item.quantity) }} {% endfilter %}</span></td>
-			</tr>
+        {% for item in model.items %}
+            {% if (not item.originalLineId) %}
+                {% if (item.quantity > 0) %}
+                    <tr class="mz-ordersummary-line mz-ordersummary-line-item">
+                        <td>
+                        {{ item.name }}
+                        <dl>
+                            <dd>{{ item.productCode }}</dd>&nbsp;
+                        </dl>
+                        </td>
+                        <td align="right"><span class="mz-item-price">{% filter currency %} {{ item.actualPrice|multiply(item.quantity) }} {% endfilter %}</span></td>
+                    </tr>
+                {% endif %}
+            {% endif %}
 		{% endfor %}
     </table>
+    
+    {% if model.substitutedItems.length > 0 %}
```

Comment:
It's been a good practice to check the object before accessing its properties.
`{% if model.substitutedItems && model.substitutedItems.length > 0 %}`